server/lib/realtime/replication.ex

```diff
@@ -38,6 +38,8 @@ defmodule Realtime.Replication do
 
   alias Realtime.SubscribersNotification
 
+  @convert_column_types ["timestamp", "timestamptz"]
```

Comment:
I also found [_timestamp](https://github.com/supabase/realtime/blob/master/server/lib/decoder/oid_database.ex#L115) and [_timestamptz](https://github.com/supabase/realtime/blob/master/server/lib/decoder/oid_database.ex#L119) in `Realtime.OidDatabase` but don't know when those types are actually used during replication.